### PR TITLE
Subsetter Updates

### PR DESCRIPTION
**Pull request by pburnsdata**
# Issues
#235 
#236 
#237

### CODE REVIEW

**Comment by pburnsdata** [comment]
I tried to make sure this would be backwards-compatible, but keep your eyes peeled for anything you think might cause something to break.

File: src/chart/draw/consolidateData.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default function consolidateData(raw) {
     if (this.filters.length) {
         this.filters.forEach(filter => {
             this.filtered_data = this.filtered_data.filter(d => {
-                return filter.val === 'All'
+                return filter.all === true && filter.index === 0
```
Comment (jwildfire) [inline src/chart/draw/consolidateData.js]: @pburnsdata Does `filter.index` change when the user updates a subsetter input? What code controls that update?

File: src/controls/makeSubsetterControl.js [hunk 2]
```diff
@@ -85,11 +96,14 @@ export default function makeSubsetterControl(control, control_wrap) {
             });
         } else {
             let value = select(this).select('option:checked').property('text');
+            let index = select(this).select('option:checked').property('index');
```
Comment (pburnsdata) [inline src/controls/makeSubsetterControl.js]: @jwildfire this is where it gets updated

File: src/chart/draw/consolidateData.js (hunk 1 above)
Comment (pburnsdata) [reply src/chart/draw/consolidateData.js]: yes, just like val changes - the code that controls it is below

File: build/webcharts.js [hunk 3]
```diff
@@ -3068,6 +3074,8 @@
             .attr('multiple', control.multiple ? true : null)
             .datum(control);
 
+        var specifiedValues = controls.values;
```
Comment (samussiah) [inline build/webcharts.js]: should be `control.values`

Comment (samussiah) [reply build/webcharts.js]: fixed in ff30e6ed91f20492048fa056deb56502daad7f32

**Pull request closed without merging**